client/asset/btc/btc.go

```diff
@@ -730,12 +732,18 @@ func openSPVWallet(cfg *BTCCloneCFG) (*ExchangeWallet, error) {
 	}
 
 	btc.node = loadSPVWallet(cfg.WalletCFG.DataDir, cfg.Logger.SubLogger("SPV"), peers, cfg.ChainParams)
+	btc.traits = asset.WalletTraitRescanner
 
 	return btc, nil
 }
 
 var _ asset.Wallet = (*ExchangeWallet)(nil)
 
+// Traits returns the traits for the type of wallet.
+func (btc *ExchangeWallet) Traits() asset.WalletTrait {
+	return btc.traits
+}
+
```

Comment:
I don't see why the wallet needs to know anything about the `asset.WalletTrait`. The wallet already knows what it's capabilities are.